packages/framework/src/Model/Mail/MailTemplateDataFactory.php

```diff
@@ -88,4 +93,33 @@ public function createFromOrderStatuses(array $orderStatuses, array $mailTemplat
 
         return $orderStatusMailTemplatesData;
     }
+
+    /**
+     * @param string|null $body
+     * @param int $domainId
+     * @return string
+     */
+    protected function ensureBodyIsInCorrectGrapeJsFormat(
+        ?string $body,
+        int $domainId = Domain::FIRST_DOMAIN_ID,
+    ): string {
+        $locale = $this->domain->getDomainConfigById($domainId)->getLocale();
+
+        if ($body === null || ctype_space($body) || strip_tags($body) === '') {
```

Comment:
Certain content (like `<div>    </div>`, for example) can still cause trouble.

what about 
```suggestion
        if ($body === null || trim(strip_tags($body)) === '') {
```